zerver/openapi/zulip.yaml

```diff
@@ -3867,6 +3867,13 @@ paths:
                 allOf:
                   - $ref: "#/components/schemas/BadEventQueueIdError"
                   - description: |
+                      #### BAD_EVENT_QUEUE_ID errors
+                      A compliant client will handle this error by re-initializing itself
+                      (e.g. a Zulip web app browser window will reload in this case).
+
+                      See [the /register endpoint docs](/api/register-queue) for details on how to
+                      handle these correctly.
+
                       If the target event queue has been garbage collected, you'll get the
                       following error response
```

Comment:
This last sentence seems awkward, but maybe that's not new.

Reply:
Modified the descriptions -- might be little better now :)